service/src/test/java/com/github/swapbook/repositories/FakeUserRepositoryTest.java

```diff
@@ -1,14 +1,17 @@
 package com.github.swapbook.repositories;
```

Comment:
wrzuć to do podfolderu com.github.swapbook.repositories.user żeby było podobnie jak specimen

Reply:
ok